packages/components/core/pop-up/constants.ts

```diff
@@ -45,3 +45,9 @@ export enum PopUpSizes {
     Normal = 'medium',
     Large = 'large'
 }
+
+/**
+ * Default value when configuring overlay in popups (Autocomplete, Datepicker, Dropdown, Select, Tags Autocomplete,
+ * Tags Input, Timezone, TreeSelect).
+ */
+export const defaultOffsetY: number = 4;
```

Comment:
```suggestion
export const POP_UP_OFFSET_Y_DEFAULT: number = 4;
```

мб константы будем аперкейсом именовать? 

можно еслинт настроить под такое...

Reply:
да, возможно в дальнейшем это станет параметром, который будет провайдится из конфига... сейчас важно его наличие, а то как называется и в каком регистре - это вкусовщина о которой можно долго спорить :)